app/schemas/models/user_exam.schema.js

```diff
@@ -0,0 +1,16 @@
+const c = require('./../schemas')
+
+const UserExamSchema = c.object({
+  title: 'UserExam',
+  description: 'Relationship between users and exams, one to many',
+  properties: {
+    userId: c.objectId({ title: 'User Id' }),
+    examId: c.objectId({ title: 'Exam Id' }),
+    startDate: c.stringDate(),
+    endDate: c.stringDate(),
+    submitted: { type: 'boolean', title: 'Submitted' },
```

Comment:
i m guessing this means completed, can we add it in description or rename field if it means that.